### PR TITLE
Globalnet support for OVN Kubernetes deployments

### PR DESCRIPTION
**Pull request by sridhargaddam**
Signed-off-by: Sridhar Gaddam <sgaddam@redhat.com>

### CODE REVIEW

File: submariner/globalnet-ovn.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+     * IPtable rules in `SUBMARINER-POSTROUTING` chain of `nat` table not to perform any NAT operation on this traffic.
+
+![SubmarinerOVNTopology](./images/ovn-submariner-architecture.svg)
+
+So, in the Submariner OVN architecture the inter-cluster traffic touches the host before leaving the Gateway node via
+the tunnel. Because of this, we can leverage the current Globalnet design which programs the iptable rules on the
+Gateway node even for OVN-Kubernetes deployments. 
+
+Naturally, a question would arise if this architecture would work going forward. 
+The answer would be yes because of the way we are creating the Submariner pipeline in OVN.
```
Comment (skitt) [inline submariner/globalnet-ovn.md]: This isn’t obvious to me — what potential problems give rise to the question? How does our method of creating the pipeline address them?

**Review by skitt — CHANGES_REQUESTED** [review]

File: submariner/globalnet-ovn.md (hunk 1 above)
Comment (sridhargaddam) [reply submariner/globalnet-ovn.md]: Please take a look at the updated description.

**Review by dfarrell07 — APPROVED** [review]

**Review by aswinsuryan — APPROVED** [review]

File: submariner/globalnet-ovn.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+* Any enhancements/fixes that go into Globalnet controller will benefit all the CNIs.
+* It will take less effort to support Globalnet for OVN-Kubernetes deployments.
+
+### Cons
+
+* Nothing in particular.
+
+### Backward Compatibility
+
+Now to support this, we require a couple of changes in Network Plugin Syncer and OVN route-agent plugin code.
```
Comment (yboaron) [inline submariner/globalnet-ovn.md:97]: Do you mean the changes mentioned above?

File: submariner/globalnet-ovn.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+supports OVN-Kubernetes it's important to fill this feature parity.
+
+## Proposal
+
+This enhancement proposes that we continue with the current design of Globalnet based on IPtables/IPsets even with
+OVN-Kubernetes. In order to get all the Globalnet features working, we will require a couple of changes in Network
+Plugin Syncer code as well as OVN Route agent handler.
+
+## Design Details
+
```
Comment (yboaron) [inline submariner/globalnet-ovn.md:20]: Excellent description,
 thanks for the detailed info!

File: submariner/globalnet-ovn.md (hunk 2 above)
Comment (sridhargaddam) [reply submariner/globalnet-ovn.md:97]: Yes, that's correct Yossi.

**Review by yboaron — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**